GliaWidgets/SecureConversations/Welcome/SecureConversations.WelcomeViewModel.swift

```diff
@@ -214,6 +215,11 @@ extension SecureConversations.WelcomeViewModel {
             return .disabled
         }
 
+        // Is there a ticket?
```

Comment:
For whom is this question?

Reply:
This is additional question for clarifying the decision to return particular button state. It is one of [many](https://github.com/salemove/ios-sdk-widgets/blob/79542a2ce8b95475c717cb466919ff9721f451d0/GliaWidgets/SecureConversations/Welcome/SecureConversations.WelcomeViewModel.swift#L197) such questions.